content/_index.fr.md

```diff
@@ -14,9 +14,13 @@ usecases:
     caption_1: utilise OpenFisca et sa
     caption_2: pour informer les citoyens de Barcelone de leur éligibilité aux aides sociales.
 
-  reforms:
+  leximpact:
     title: Déterminer l’impact de réformes
     description: Comparez la loi telle qu’elle est et telle qu’elle pourrait être.
+    caption_1: utilise OpenFisca et sa
+    caption_2: pour simuler, de façon rapide, l'impact des réformes paramétriques de l'impôt sur le revenu.
```

Comment:
```suggestion
    caption_2: pour simuler, de façon rapide, l'impact des réformes paramétriques de l'impôt sur le revenu sur des ménages.
```
I mean, add on who/what we are calculating an impact to help someone discovering openfisca here?

Reply:
I think what you calculate income tax on are tax households, that is in the French legislation _foyers fiscaux_, rather than just households, or _ménages (fiscales)_.

If the intended audience for your suggestion is someone who would like to use OpenFisca to calculate income tax, it would in my opinion be misleading to tell them OpenFisca allows them to do it on _ménages_ (people living in the same dwelling) and not, as it's what they'd expect, on _foyers fiscaux_ (people paying their taxes together).

On the other hand, if the audience is more generalist, _ménage_ would be ok (anyway I'm not French, so I do not know what do people perceive by _ménage_).

What do you think?

Reply:
I proposed a new wording, let me know what do you think!